topsail/testing/regression.py

```diff
@@ -0,0 +1,168 @@
+import os
+import json
+
+import numpy as np
+from functools import reduce
+
+from opensearchpy import OpenSearch
+
+regression_return_codes = {
+    "POS_REG": 1,
+    "NEG_REG": -1,
+    "NONE": 0,
+    "UNDEF": 2,
+}
+
+def get_from_path(d, path):
+    return reduce(dict.get, path.split("."), d)
+
+class RegressionIndicator:
+
+    def __init__(
+            self,
+            client,
+            kpi,
+            settings=None,
+            combine_func=None,
+            x_var="_source.@timestamp",
+            y_var="_source.value",
```

Comment:
I'm not sure to understand how (now where) this is used? 🤔 
`_source....` should be internal OpenSearch, this module should not receive OpenSearch values